syne_tune/optimizer/schedulers/multiobjective/multi_surrogate_multi_objective_searcher.py

```diff
@@ -75,7 +75,7 @@ def __init__(
         estimators: Dict[str, Estimator],
         mode: Optional[List[str]] = None,
         points_to_evaluate: Optional[List[Dict[str, Any]]] = None,
-        scoring_class_and_args: Optional[ScoringClassAndArgs] = None,
```

Comment:
I think the right typing is actually: `Callable[[Any], ScoringFunction]` since we are passing a constructor to the scoring function not an instance of the scoring function

Reply:
I see, good point